source/reference/limitations.txt

```diff
@@ -0,0 +1,73 @@
+.. _c2c-limitations:
+
+===========
+Limitations
+===========
+
+.. include:: /includes/preview-warning
+
+{+c2c-full-product-name+} is under active development. This preview
+release has several limitations.
+
+General Limitations
+-------------------
+
+- The minimum supported version is MongoDB 6.0.
+- The minimum support Feature Compatibility Version is 6.0. 
+- MongoDB Community Edition is not supported.
+- The source and destination clusters must have the same release
+  version.
+- Environmental assumptions are not validated as pre-checks. 
+- The Primary cluster cannot be changed while syncing.
+- If write blocking is disabled,the client must prevent writes to the
+  source cluster after starting the commit process.
+- DDL operations are not supported during syncing. 
```

Comment:
Can we indicate which limitations will be for GA and which for preview where we know. I'd love to indicate that at GA DDL will be supported

Reply:
I don't have that info, maybe @alan-zheng-mdb or @ldennis knows.

We probably shouldn't promise anything that specific anyway to avoid customer reliance issues in case something slips.

Reply:
I think there are some features we must support by GA, so I think I'm comfortable saying they'll come for GA. @ldennis and @alan-zheng-mdb , what do you think?